hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/directory_markers.md

```diff
@@ -161,7 +176,7 @@ When a file is created under a path, the directory marker is deleted. And when a
 file is deleted, if it was the last file in the directory, the marker is
 recreated.
 
-And, historically, When a path is listed, if a marker to that path is found, *it
+And, historically, when a path is listed, if a marker to that path is found, *it
 has been interpreted as an empty directory.*
```

Comment:
(This isn't added in this PR but...) is this really true?

I tried an integ test using `listFiles` on the Hadoop 3.0 code base. It seemed happy. Is it worth being specific with what will or won't make this assumption?

Reply:
it is for some specific codepaths which do a probe which explicitly looks for empty dirs, rm and mv in particular.

Reply:
ACK.

For public documentation, probably makes sense to keep it as is. It was just a little confusing when digging into the detail.